### DISCUSSION TITLE
dag dependencies dosen't apear when using TriggerDagRunOperator as a dynamic task

### DISCUSSION THREAD

**Discussion by The-Shinigami** [apache/airflow#31773] [General]
### Apache Airflow version

2.6.1

### What happened

im using TriggerDagRunOperator as a dynamic task to execute a number of dags depending in the input , but dag dependencies doesn't appear in the dag dependencies view

### What you think should happen instead

the dag dependencies to apear

### How to reproduce

create a dag with TriggerDagRunOperator as a dynamic task for a other dag, and go to the dependencies view

### Operating System

windows

### Versions of Apache Airflow Providers

v2.6.1

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
There is nt enough informaion/log/ screenshots to do anything with the issue. In order for anyone to take a look, more information and circumstances and details would be needed, Converted to discussion for now.
